test/test_katello/test_agent/test_pulp/test_libdnf.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import unittest
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../src/'))
+
+try:
+    from katello.agent.pulp import libdnf
+except ImportError:
+    print("Dnf wasn't present")
+
+try:
```

Comment:
what's this try block for?

Reply:
oh because mock.patch and mock.MagicMock are not available in python 2. Thankfully all dnf  tests are in python 3.. 
I could ve prolly said "if python3" import. I went with the try catch route instead